ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs

```diff
@@ -8,11 +8,6 @@ static class ViewManagersPropertyCache
     {
         private static readonly IReadOnlyDictionary<string, IPropertySetter> s_shadowEmpty = new Dictionary<string, IPropertySetter>();
 
-        private static readonly object[] s_viewArgsBuffer = new object[2];
```

Comment:
I'd rather wait to do this when that next set of UIManager and View files are moved to Shared. I did a quick look and I/we can do PropertySetter and the other dependencies once the current set of outstanding PRs are moved.